test/lit/lookup_in_type.test

```diff
@@ -0,0 +1,18 @@
+RUN: %clangxx -g -o %t lookup_in_type/test_program.cc
+RUN: %lldb -b \
+RUN:       -o 'script import gala_compatibility' \
```

Comment:
I like writing the script part of this kind of unit test as something I can also run from gdb (manually, I haven't set up any way to run them both and compare the outputs, but it's still useful to be able to check what gdb does).

Reply:
That sounds reasonable, but I'm not sure what that means in this case. For example, would it be better if I put all the code in a single python file (that can then be executed with a single command)? The execute command would still be different in lldb vs. gdb, but at least it would be single cmd (?)

Reply:
Yeah, that's why many of the tests are python modules like https://github.com/sivachandra/gala/blob/master/test/lit/simple/__init__.py

That way I can run `PYTHONPATH=$GALA_ROOT/test/lit gdb -ex 'py import whatever'` and it will run the whole test. Not very important, but nice to have sometimes. I'm sure there are better options, this was just a habit I adopted in a pinch while trying to accomplish something else :)